server/routes/user_screen/fetchById.js

```diff
@@ -0,0 +1,15 @@
+const userScreen = require("../../query/user_screen");
+const express = require("express");
+const router = express.Router();
+
+router.get("/:id", async (req, res) => {
+  try {
+    const id = req.params.id;
```

Comment:
Do we need to handle where case `id` is `null` and or not empty (ex. "") ?

Reply:
if I do "http://localhost:8000/user_screen/id/" it just throws an error (Cannot GET /user_screen/id/), so I don't think an id of "" is a problem.

since the ID comes from the route, I'm not sure it's possible for the id to be null.
if I do "http://localhost:8000/user_screen/id/null" it throws an error on the server and the response hangs. it should definitely still return a response with the error (invalid UUID or whatever), so i'll fix that and redo the PR